### ISSUE TITLE
Feature Request: Display images in messages

### ISSUE THREAD

**Issue by arena** [ProgressiveCoders/Spoke#1354]
As a texter, when somebody I've been texting sends me a photo of their "I voted!" sticker, I want to see it. :)

Solution:
Display images

**Comment by williamtheaker** [comment]
I'm worried that creeps would send inappropriate pictures and put volunteers in uncomfortable situations.

**Comment by arena** [comment]
In my experience, this hasn't happened. But there are ways to accommodate your fear other than not allowing images at all. E.g. a step to reveal the the image by clicking (which can be ignored)

**Comment by arena** [comment]
slow reveal might be a nice feature to add to this as well .. bunch of pixels, more clear, more clear, ..at any point you can tap to stop the transition to clarity. ..if it's going to be weird at least you don't see the details.

**Comment by DinahSanders** [comment]
There were lengthy discussions among the MoveOn Text Team leadership about whether to allow images. Can't remember if it was 2017 or 2018 or early 2019 (or all three). Erik is the one to talk to about this.

Harassment via sexual or violent imagery is a real concern in preserving volunteer texter mental health (and, frankly, motivation to keep volunteering).

**Comment by arena** [comment]
One of the things I really enjoyed while texting with Hustle was getting "I voted" stickers texted to me after lengthy persuasion convos. The last time I did GOTV texting with Spoke people sent me images I couldn't see and confirmed that they were "I voted" stickers when I asked them. I, personally, find it more disappointing to miss good images than to screen bad images. I think it's a design challenge that is do-able. To transmit images in a way that makes them optional and deliberate to view by volunteers. For instance, I could choose to not view an image from a person I just sent a first text to. But would definitely want to see an image from a person who I had done some deep canvassing with on election day.
